### PR TITLE
fix(integration-test): fixes to stabilize integration tests

### PR DESCRIPTION
**Pull request by fruch**
during working on #5585, we needed to run those test on a weaker machines, and those
are the result of that work
  
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by soyacz — APPROVED** [review]
LGTM

File: unit_tests/test_version_utils.py [hunk 1]
```diff
@@ -397,7 +397,7 @@ def test_scylla_version_grouped_regexp(full_version, version, date, commit_id):
 @pytest.mark.parametrize("version, expected", (
     ("4.6.rc2-20220102.e8a1cfb6f", "scylladb/scylla:4.6.rc2"),
     ("5.0.1-0.20220719.b177dacd3", "scylladb/scylla:5.0.1"),
-    ("5.1", "scylladb/scylla:latest"),
+    ("5.1", "scylladb/scylla:5.1"),
```
Comment (vponomaryov) [inline unit_tests/test_version_utils.py:400]: Need to add following:

```
("5.2", "scylladb/scylla:latest"),
```

**Review by vponomaryov — CHANGES_REQUESTED** [review]

File: unit_tests/test_version_utils.py (hunk 1 above)
Comment (fruch) [reply unit_tests/test_version_utils.py:400]: why ? for it to fail once they would start the 5.2 release ?
I think this is a bit pointless

Comment (vponomaryov) [reply unit_tests/test_version_utils.py:400]: I know that we will hit the same problem again, but it is coverage of the functionality.
We either test that the next version is refering to "latest" one or not.

Comment (fruch) [reply unit_tests/test_version_utils.py:400]: right now 5.2 doesn't exist, we already have a test covering that.

I'm don't understand what adding this new test would help us ?

Comment (vponomaryov) [reply unit_tests/test_version_utils.py:400]: It is for testing mapping of next unavilable version yet and `latest` tag.
If you think we do not need then ok.

**Review by vponomaryov — APPROVED** [review]

**Pull request merged**